components/boloforms/sources/new-response-instant/new-response-instant.mjs

```diff
@@ -0,0 +1,39 @@
+import common from "../common/base.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  ...common,
+  key: "boloforms-new-response-instant",
+  name: "New Response (Instant)",
+  description: "Emit new event when a filled form response is received.",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    formId: {
+      propDefinition: [
+        common.props.boloforms,
+        "formId",
+      ],
+      withLabel: true,
+    },
+  },
+  methods: {
+    ...common.methods,
+    getWebhookEvent() {
+      return "new_form_response";
+    },
+    getDocs() {
+      return this.formId;
```

Comment:
Is `withLabel` is needed for `formId`? If not, it can be removed. Otherwise, `this.formId` will be an object with label & value properties, so you'd need to add `.value` here.
```suggestion
      return this.formId.value;
```

Reply:
the endpoint expects exactly an object with value and label, so it's right.